packages/analytics/src/tune.js

```diff
@@ -23,33 +23,81 @@ const methods = {
 const allMethods = Object.keys(methods).join(',');
 
 /**
- * Take all `Object` and sort them with selected field
+ * Create and replace the id with a unify id that can be used with [sort](#sort)
  *
- * ```json
- * [{
- * }]
- * ```
+ * Créer et remplacer l'identifiant par un identifiant unifié qui peut être utilisé avec [sort](#sort)
  *
- * Script:
+ * ### Example / Exemple
+ *
+ * #### Script / Scénario
  *
  * ```ini
+ * ; Import analytics plugin required to use tune
+ * ; Importation du plugin analytique nécessaire pour utiliser tune
  * [use]
  * plugin = analytics
  *
+ * ; Using "tune" with default settings
+ * ; Utilisation de "tune" avec les paramettre par defaut
```

Comment:
<!-- Typo comments are like todo:, where the main issue is a mispelling.  -->
**typo:** paramètres & défaut

```suggestion
 * ; Utilisation de "tune" avec les paramètres par défaut
```